src/components/UI/ReadMore/ReadMore.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+import styles from './ReadMore.module.scss';
+import {readMoreTxt} from './ReadMore.strings';
+
+export const ReadMore = ({URL, text, openInNewTab = false}) => {
+  return (
+    <a className={styles.readMore} href={URL} rel="noreferrer" target={openInNewTab && '_blank'}>
+      {text || readMoreTxt}
```

Comment:
No need this statement, you can define default value in the prop list
`text = readMoreTxt`